Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
             targets: ["Metaplex"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/metaplex-foundation/Solana.Swift.git", branch: "1.3.0"),
+        .package(url: "https://github.com/metaplex-foundation/Solana.Swift.git", branch: "feature/allow-different-accounts"),
```

Comment:
This is temporary and will be replaced with the new version of `Solana.Swift` once it is merged.